### PR TITLE
fix(binance) - fixed initial margin bug and also parahraph in docs explaining things

### PR DESCRIPTION
**Pull request by ttodua**
for example, opened a positon with `0.57`$ margin, and later added 1$ onto margin.

.fetchPositions (['GAS/USDT:USDT']) 

```
[
  {
    info: {
      symbol: "GASUSDT",
      positionAmt: "-1.0",
      entryPrice: "8.616",
      breakEvenPrice: "8.611692",
      markPrice: "8.73600000",
      unRealizedProfit: "-0.12000000",
      liquidationPrice: "10.03654386",
      leverage: "15",
      maxNotionalValue: "37500",
      marginType: "isolated",
      isolatedMargin: "1.45109202",
      isAutoAddMargin: "false",
      positionSide: "BOTH",
      notional: "-8.73600000",
      isolatedWallet: "1.57109202",
      updateTime: "1700147205685",
      isolated: true,
      adlQuantile: "2",
    },
    id: undefined,
    symbol: "GAS/USDT:USDT",
    contracts: 1,
    contractSize: 1,
    unrealizedPnl: -0.12,
    leverage: 15,
    liquidationPrice: 10.03654386,
    collateral: 1.45109202,
    notional: 8.736,
    markPrice: 8.736,
    entryPrice: 8.616,
    timestamp: 1700147205685,
    initialMargin: 0.58240002,
    initialMarginPercentage: 0.06666667,
    maintenanceMargin: 0.13104,
    maintenanceMarginPercentage: 0.015,
    marginRatio: 0.0903,
    datetime: "2023-11-16T15:06:45.685Z",
    marginMode: "isolated",
    marginType: "isolated",
    side: "short",
    hedged: false,
    percentage: -20.6,
    stopLossPrice: undefined,
    takeProfitPrice: undefined,
  },
]
```

### CODE REVIEW

**Comment by ttodua** [comment]
ready to review

File: ts/src/binance.ts [hunk 1]
```diff
@@ -7373,7 +7373,10 @@ export default class binance extends Exchange {
         if (!rational) {
             initialMarginPercentageString = Precise.stringAdd (initialMarginPercentageString, '1e-8');
         }
-        const initialMarginString = Precise.stringDiv (Precise.stringMul (notionalStringAbs, initialMarginPercentageString), '1', 8);
+        let initialMarginString = this.safeString (position, 'isolatedWallet');
```
Comment (ttodua) [inline ts/src/binance.ts:7376]: There is field `isolatedWallet` : https://binance-docs.github.io/apidocs/futures/en/#position-information-v2-user_data:~:text=%220%22%2C%2C-,%22isolatedWallet%22%3A%20%220%22,-%2C%0A%20%20%20%20%20%20%20%20%22symbol%22
that carries that value, soo we dont need calculations here

Comment (carlosmiei) [reply ts/src/binance.ts:7376]: We might consider this change, but I'm not sure how this fixes the pointed bug. That bug might appear elsewhere if the difference is in some Precise function

Comment (ttodua) [reply ts/src/binance.ts:7376]: no, maybe i didnt express it well. this does not fix the aforementioned issue we discussed on discord. this is just an additional bugfix (enhancement) inside position parsing specifically for binance, that i've found down the road. 
so, this PR title might have been different, sorry.

Comment (ttodua) [reply ts/src/binance.ts:7376]: wait, fixing some issue in this PR

Comment (ttodua) [reply ts/src/binance.ts]: done. now ready to review

**Comment by carlosmiei** [comment]
@ttodua Yeah maybe initial margin is a bit ambiguous, especially if it includes margin that was added after the opening of the position but yeah it's out of the scope of this PR

**Comment by frosty00** [comment]
@ttodua that's collateral not initial margin.

**Comment by frosty00** [comment]
initialMargin = minimum amount of margin to keep position open at a given leverage.

collateral = current margin in position including unrealised pnl and realized pnl

**Comment by ttodua** [comment]
> @ttodua that's collateral not initial margin.

@frosty00 it's not collateral. 
I mean, `isolatedWallet` is not collateral, but `isolatedMargin` is the collateral there. that's what I made a PR. 
l

**Comment by ttodua** [comment]
I will recheck this PR

**Pull request closed without merging**